specification/batch/data-plane/Microsoft.Batch/stable/2018-03-01.6.1/BatchService.json

```diff
@@ -11546,7 +11546,7 @@
             "$ref": "#/definitions/ResourceFile"
           },
           "title": "A list of files that the Batch service will download to the compute node before running the command line.",
-          "description": "Files listed under this element are located in the task's working directory."
+          "description": "Files listed under this element are located in the task's working directory. There is a maximum size for the list of resource files.  When this is exceeded, the task cannot be added and the response error code will be RequestEntityTooLarge. If this occurs, the collection of ResourceFiles must be reduced in size. Alternatives include using .zip files, the Application Package feature, or Docker Containers."
```

Comment:
"When this is exceeded, the task cannot be added and the response error code will be RequestEntityTooLarge." -> "When the max size is exceeded, the request including task will fail and the response error code will be RequestEntityTooLarge."

"If this occurs, the collection of ResourceFiles must be reduced in size. Alternatives include using .zip files, the Application Package feature, or Docker Containers." -> "If this occurs, the collection of ResourceFiles must be reduced in size. This can be achieved using .zip files, Application Packages, or Docker Containers."?

Reply:
"When the max size is exceeded, the request including task will fail and the response error code will be RequestEntityTooLarge." => "When the max size is exceeded, the request will fail and the response error code will be RequestEntityTooLarge."  --the task bit seems to be inferred by request failure. Saying 'task will fail' seems to indicate that we will still do it and it will fail to me

"If this occurs, the collection of ResourceFiles must be reduced in size. This can be achieved using .zip files, Application Packages, or Docker Containers." ==> "If this occurs, the collection of ResourceFiles must be reduced in size. This can be achieved using .zip files, Application Packages, or Docker Containers." -- I like the removal of feature, sounds cleaner.

Reply:
no change in the second block i liked yours